portfolio/js/controllers/portfolioController.js

```diff
@@ -0,0 +1,10 @@
+(function(module){
+    // Make a controller object, give it a show method, pass it to the global
+    var portfolioController = {};
+
+    portfolioController.showPortfolio = function() {
+        $('.tab-content').hide();
+        $('.tab-content').filter('#projects').fadeIn('slow');
```

Comment:
Turns out, the filtering is necessary because the previously hidden articles don't show up when this ID is selected!